dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +161,63 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactDQMSolver():
+    """A simple exact solver for testing and debugging code using your local CPU.
+
+    Notes:
+        This solver becomes slow for problems with 18 or more
+        variables, worse for increasingly many cases per variable.
+
+    """
+    properties = None
+    parameters = None
+
+    def __init__(self):
+        self.properties = {}
+        self.parameters = {}
+
+    def remove_unknown_kwargs(self, **kwargs):
```

Comment:
I think `remove_unknown_kwargs = Sampler.remove_unknown_kwargs` should work fine.